src/service/reception-service.js

```diff
@@ -0,0 +1,41 @@
+const jwt = require("jsonwebtoken");
+const Receptions = require("../models/reception-model");
+const config = require("../../config");
+
+class ReceptionService {
+  async getList(token) {
+    const findUser = jwt.verify(token, config.jwtAccess);
+    const receptionList = Receptions.find({
+      user_id: findUser.id,
+    });
+    return receptionList;
+  };
+
+  async createList(name, doctor, date, complaint, token) {
+    const findUser = jwt.verify(token, config.jwtAccess);
+    const newList = await Receptions.create({
+      name: name,
+      doctor: doctor,
+      date: date,
+      complaint: complaint,
+      user_id: findUser.id,
+    });
+    const result = Receptions.find({ user_id: findUser.id });
+    return result;
+  };
+
+  async updateList(body, token) {
+    const {_id} = body;
+    const findUser = jwt.verify(token, config.jwtAccess);
+    const editList = await Receptions.updateOne({ _id: _id }, body);
+    const result = Receptions.find({ user_id: findUser.id });
```

Comment:
This line is not needed
Again, return editList and replace the updateOne method with findOneAndUpdate

otherwise why do you need editList?